website/versioned_docs/version-14.0/controlling-fixtures/using-the-select-buttons-and-wheels.md

```diff
@@ -216,45 +216,29 @@ To go back to normal, press \<Unfold\> then \[Exit Unfold\].
 You can also use the numeric keypad to quickly select cells. The syntax
 is as follows: (\<THRO\> is called Through on some consoles)
 
-  .            all sub-fixtures of selection                    
 
-  n\.            all sub-fixtures of fixture n
-
-  . \<THRO\> .j    sub-fixtures 1j of all selected fixtures         
-
-  n\. \<THRO\>       all sub-fixtures of fixture nlast consecutive of type
-
-  n \<THRO\> .j      shorthand for above                              
-
-  n\. \<THRO\> i     sub-fixtures 1i of fixture n
-
-  .m           sub-fixture m of all selected fixtures           
-
-  n\. \<THRO\> i.j   sub-fixtures 1j of fixtures ni
-
-  .m \<THRO\>      sub-fixtures mlast of all selected fixtures      
-
-  n.m            sub-fixture m of fixture n
-
-  .m \<THRO\> .j   sub-fixtures mj of all selected fixtures         
-
-  n.m \<THRO\>       sub-fixtures mlast of fixture n
-
-  .m \<THRO\> j    shorthand for above                              
-
-  n.m \<THRO\> i     sub-fixtures mi of fixture n
-
-  n \<THRO\> i.    all sub-fixtures of fixtures ni                  
-
-  n.m \<THRO\> i.    sub-fixture mlast of fixtures ni
-
-  n \<THRO\> i.j   sub-fixture j of fixtures ni                     
-
-  n.m \<THRO\> i.j   sub-fixtures mj of fixtures ni
-
-  n \<THRO\> .j    sub-fixture 1j of fixture n                      
-
-  n.m \<THRO\> .j    sub-fixtures mj of fixture n
+|button presses         |selection                      |
+|-----------------------|-------------------------------|
+|\<.\>                   |all sub fixtures of selection  |                    
+|n \<.\>                |all sub fixtures of fixture n  |
```

Comment:
````suggestion
| **n** \<.\>                |all sub fixtures of fixture **n**  |
````
I think capitalise the letters like this would make it clearer